demos/index.php

```diff
@@ -58,10 +58,23 @@
 // list all filestore files
 $c2 = $columnsLayout->addColumn();
 Header::addTo($c2, ['All Filestore Files']);
-$gr = Grid::addTo($c2, [
+$gr = Crud::addTo($c2, [
     'paginator' => false,
 ]);
 $files = new File($app->db, ['flysystem' => $filesystem]);
+$files->removeUserAction('add');
```

Comment:
I am not completely sold with CRUD here, Grid cannot have buttons?

Reply:
Why not Crud? It supports UserActions.
Even if we can do similar thing with Grid, then still - why not do the same with Crud?

Reply:
CRUD is for editing which is not what we need here. If not easy with Grid, I am ok with CRUD.

Reply:
ok, changed to use grid instead